packages/base/cards-grid.gts

```diff
@@ -61,17 +61,20 @@ class Isolated extends Component<typeof CardsGrid> {
       </ul>
 
       {{#if @context.actions.createCard}}
-        <div class='cards-grid__add-button'>
-          <IconButton
-            @icon='icon-plus-circle'
-            @width='40px'
-            @height='40px'
-            @tooltip='Add a new card to this collection'
-            @tooltipPosition='left'
-            class='add-button'
-            {{on 'click' this.createNew}}
-            data-test-create-new-card-button
-          />
+        <div class='cards-grid__add-button'> 
+          <Tooltip 
+            @content='Add a new card to this collection'
```

Comment:
In my experience sooner or later comes a need to be able to have content more complicated than a line of text. So I think if we're at it right now it would be really good to add separate blocks for trigger and content.

Something like: 

```hbs
 <Tooltip>
   <:trigger>
     <IconButton />
   </:trigger>
   <:content>
     Add a new card to this collection
   </:content>
</Tooltip>
```

Reply:
Updated https://github.com/cardstack/boxel/pull/477/commits/1ce6117529b1f4238cc1f7a95896f058b2c737bc